web-api/enable-deploying-dynamo-stream-trigger.sh

```diff
@@ -7,3 +7,8 @@
 
 UUID=$(aws lambda list-event-source-mappings --function-name "arn:aws:lambda:us-east-1:${AWS_ACCOUNT_ID}:function:streams_${ENV}_${DEPLOYING_COLOR}" --region us-east-1 | jq -r ".EventSourceMappings[0].UUID")
 aws lambda update-event-source-mapping --uuid "${UUID}" --region us-east-1 --enabled
+
+STATE=$(aws lambda get-event-source-mapping --uuid "${UUID}" --region us-east-1 --query "State")
+echo "${ENV}'s ${DEPLOYING_COLOR} dynamo stream is now: ${STATE}"
+
+sleep 90
```

Comment:
I remember when this was done that AWS reports its mapping is updated immediately after you set it even though it does take some time to take actual effect. And I suppose that `sleep 90` is representing that time here.

Still, I wonder if we should print the STATE after the sleep since that's what I think this informational output is trying to report? 

And please, include a newline at the end.